cli/src/tool/templates/rust/main.rs.jinja

```diff
@@ -17,7 +17,9 @@ enum Output {
     Ok {
         // Add output ports for the `Ok` variant
     },
-    // Add more output variants if needed
+    Err {
+        reason: String,
```

Comment:
let's also add a `code` to the template to suggest as good practice

Reply:
@aelesbao Something like this? https://github.com/Talus-Network/nexus-sdk/pull/160/commits/ecd7499c30a726edfd1038bb0a582d39cc021cfb